.php_cs.dist

```diff
@@ -82,10 +82,15 @@ $config = PhpCsFixer\Config::create()
 // special handling of fabbot.io service if it's using too old PHP CS Fixer version
 if (false !== getenv('FABBOT_IO')) {
     try {
-        PhpCsFixer\FixerFactory::create()
-            ->registerBuiltInFixers()
-            ->registerCustomFixers($config->getCustomFixers())
-            ->useRuleSet(new PhpCsFixer\RuleSet($config->getRules()));
+        $config = PhpCsFixer\FixerFactory::create()
```

Comment:
You are overriding the `Config` instance in `$config` with a `FixerFactory` instance. Is that expected? I think this is the reason fabbot failed to run on your PR.

Reply:
Yeah, must been tired that day. Fixed.